app/executor.php

```diff
@@ -235,9 +233,7 @@ function getStorageDevice($root): Device {
                 throw new Exception('Failed to create build container', 500);
             }
 
-            if (!empty($network)) {
-                $orchestration->networkConnect($runtimeId, $network);
-            }
+            $orchestration->networkConnect($runtimeId, App::getEnv('_APP_EXECUTOR_RUNTIME_NETWORK', 'appwrite_runtimes'));
```

Comment:
in this case, would it make more sense to use an "Open Runtime" prefix? This part is planned to be an independent docker image under the OPR org.

Reply:
Renamed to `OPEN_RUNTIMES_NETWORK`. WDYT?